src/types/types.ts

```diff
@@ -76,6 +76,8 @@ export interface EdgeIo {
   /**
    * This is like `fetch`, but will try to avoid CORS limitations
    * on platforms where that may be a problem.
+   *
+   * @deprecated - Use `fetch` instead with `avoidCors: 'always'` option.
```

Comment:
Actually `avoidCors: 'auto'` would be the correct equivalent. We could actually simplify the message to `@deprecated Use EdgeIo.fetch instead, which now includes CORS avoidance by default`